diff --git a/third_party/marl/BUILD.bazel b/third_party/marl/BUILD.bazel
index 297fd4464..c63a71556 100644
--- a/third_party/marl/BUILD.bazel
+++ b/third_party/marl/BUILD.bazel
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.

+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
+
 config_setting(
     name = "linux_x86_64",
     constraint_values = [